ui/src/common/SongTitleField.js

```diff
@@ -25,14 +18,7 @@ const useStyles = makeStyles({
 })
 
 export const SongTitleField = ({ showTrackNumbers, ...props }) => {
-  const theme = useTheme()
```

Comment:
Why did you remove the logic from here and spread it into other components (breaking the behaviour for other song lists)? Why not just extending this component?